### ISSUE TITLE
Build error with makechrootpkg

### ISSUE THREAD

**Issue by bionade24** [muesli/telephant#55]
Trying to rebuild telephant-git hasn't worked out, I got some qt go bindings related error:
```
# github.com/therecipe/qt/core
core-minimal.cpp: In function ‘long long int QLineF_Intersect(void*, void*, void*)’:
core-minimal.cpp:3094:115: warning: ‘QLineF::IntersectType QLineF::intersect(const QLineF&, QPointF*) const’ is deprecated: Use intersects() instead [-Wdeprecated-declarations]
 3094 |  return static_cast<QLineF*>(ptr)->intersect(*static_cast<QLineF*>(line), static_cast<QPointF*>(intersectionPoint));
      |                                                                                                                   ^
In file included from /usr/include/qt/QtCore/QLine:1,
                 from core-minimal.cpp:36:
/usr/include/qt/QtCore/qline.h:256:19: note: declared here
  256 |     IntersectType intersect(const QLineF &l, QPointF *intersectionPoint) const;
      |                   ^~~~~~~~~
core-minimal.cpp: In function ‘int QTime_Restart(void*)’:
core-minimal.cpp:4901:43: warning: ‘int QTime::restart()’ is deprecated: Use QElapsedTimer instead [-Wdeprecated-declarations]
 4901 |  return static_cast<QTime*>(ptr)->restart();
      |                                           ^
In file included from /usr/include/qt/QtCore/qcborvalue.h:44,
                 from /usr/include/qt/QtCore/QCborValue:1,
                 from core-minimal.cpp:19:
/usr/include/qt/QtCore/qdatetime.h:229:54: note: declared here
  229 |     QT_DEPRECATED_X("Use QElapsedTimer instead") int restart();
      |                                                      ^~~~~~~
core-minimal.cpp: In function ‘void QTime_Start(void*)’:
core-minimal.cpp:4911:34: warning: ‘void QTime::start()’ is deprecated: Use QElapsedTimer instead [-Wdeprecated-declarations]
 4911 |  static_cast<QTime*>(ptr)->start();
      |                                  ^
In file included from /usr/include/qt/QtCore/qcborvalue.h:44,
                 from /usr/include/qt/QtCore/QCborValue:1,
                 from core-minimal.cpp:19:
/usr/include/qt/QtCore/qdatetime.h:228:55: note: declared here
  228 |     QT_DEPRECATED_X("Use QElapsedTimer instead") void start();
      |                                                       ^~~~~
github.com/therecipe/qt/qml
github.com/therecipe/qt/internal/binding/runtime
# github.com/therecipe/qt/internal/binding/runtime
moc.cpp: In function ‘int helper7390ef_helper7390ef_QmlRegisterType()’:
moc.cpp:76:39: warning: ‘int qmlRegisterType() [with T = helper7390ef]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
   76 |  return qmlRegisterType<helper7390ef>();
      |                                       ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:24:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp:76:39: warning: ‘int qmlRegisterType() [with T = helper7390ef]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
   76 |  return qmlRegisterType<helper7390ef>();
      |                                       ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:24:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
github.com/therecipe/qt/gui
github.com/therecipe/qt/quickcontrols2
github.com/muesli/telephant
# github.com/muesli/telephant
moc.cpp: In function ‘int ConfigBridge722e73_ConfigBridge722e73_QmlRegisterType()’:
moc.cpp:1156:45: warning: ‘int qmlRegisterType() [with T = ConfigBridge722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 1156 |  return qmlRegisterType<ConfigBridge722e73>();
      |                                             ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp:1156:45: warning: ‘int qmlRegisterType() [with T = ConfigBridge722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 1156 |  return qmlRegisterType<ConfigBridge722e73>();
      |                                             ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp: In function ‘int AccountBridge722e73_AccountBridge722e73_QmlRegisterType()’:
moc.cpp:1841:46: warning: ‘int qmlRegisterType() [with T = AccountBridge722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 1841 |  return qmlRegisterType<AccountBridge722e73>();
      |                                              ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp:1841:46: warning: ‘int qmlRegisterType() [with T = AccountBridge722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 1841 |  return qmlRegisterType<AccountBridge722e73>();
      |                                              ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp: In function ‘int Attachment722e73_Attachment722e73_QmlRegisterType()’:
moc.cpp:2001:43: warning: ‘int qmlRegisterType() [with T = Attachment722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 2001 |  return qmlRegisterType<Attachment722e73>();
      |                                           ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp:2001:43: warning: ‘int qmlRegisterType() [with T = Attachment722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 2001 |  return qmlRegisterType<Attachment722e73>();
      |                                           ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp: In function ‘int MessageModel722e73_MessageModel722e73_QmlRegisterType()’:
moc.cpp:2256:45: warning: ‘int qmlRegisterType() [with T = MessageModel722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 2256 |  return qmlRegisterType<MessageModel722e73>();
      |                                             ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp:2256:45: warning: ‘int qmlRegisterType() [with T = MessageModel722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 2256 |  return qmlRegisterType<MessageModel722e73>();
      |                                             ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp: In function ‘int Pane722e73_Pane722e73_QmlRegisterType()’:
moc.cpp:3026:37: warning: ‘int qmlRegisterType() [with T = Pane722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 3026 |  return qmlRegisterType<Pane722e73>();
      |                                     ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp:3026:37: warning: ‘int qmlRegisterType() [with T = Pane722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 3026 |  return qmlRegisterType<Pane722e73>();
      |                                     ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp: In function ‘int PaneModel722e73_PaneModel722e73_QmlRegisterType()’:
moc.cpp:3271:42: warning: ‘int qmlRegisterType() [with T = PaneModel722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 3271 |  return qmlRegisterType<PaneModel722e73>();
      |                                          ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp:3271:42: warning: ‘int qmlRegisterType() [with T = PaneModel722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 3271 |  return qmlRegisterType<PaneModel722e73>();
      |                                          ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp: In function ‘int ProfileBridge722e73_ProfileBridge722e73_QmlRegisterType()’:
moc.cpp:4391:46: warning: ‘int qmlRegisterType() [with T = ProfileBridge722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 4391 |  return qmlRegisterType<ProfileBridge722e73>();
      |                                              ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp:4391:46: warning: ‘int qmlRegisterType() [with T = ProfileBridge722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 4391 |  return qmlRegisterType<ProfileBridge722e73>();
      |                                              ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp: In function ‘int UIBridge722e73_UIBridge722e73_QmlRegisterType()’:
moc.cpp:4647:41: warning: ‘int qmlRegisterType() [with T = UIBridge722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 4647 |  return qmlRegisterType<UIBridge722e73>();
      |                                         ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp:4647:41: warning: ‘int qmlRegisterType() [with T = UIBridge722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 4647 |  return qmlRegisterType<UIBridge722e73>();
      |                                         ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp: In function ‘int AttachmentModel722e73_AttachmentModel722e73_QmlRegisterType()’:
moc.cpp:4892:48: warning: ‘int qmlRegisterType() [with T = AttachmentModel722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 4892 |  return qmlRegisterType<AttachmentModel722e73>();
      |                                                ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp:4892:48: warning: ‘int qmlRegisterType() [with T = AttachmentModel722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 4892 |  return qmlRegisterType<AttachmentModel722e73>();
      |                                                ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp: In function ‘int Message722e73_Message722e73_QmlRegisterType()’:
moc.cpp:5662:40: warning: ‘int qmlRegisterType() [with T = Message722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 5662 |  return qmlRegisterType<Message722e73>();
      |                                        ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
moc.cpp:5662:40: warning: ‘int qmlRegisterType() [with T = Message722e73]’ is deprecated: Use qmlRegisterAnonymousType instead [-Wdeprecated-declarations]
 5662 |  return qmlRegisterType<Message722e73>();
      |                                        ^
In file included from /usr/include/qt/QtQml/qqmlengine.h:47,
                 from /usr/include/qt/QtQml/QQmlEngine:1,
                 from moc.cpp:33:
/usr/include/qt/QtQml/qqml.h:138:74: note: declared here
  138 | QT_DEPRECATED_VERSION_X_5_14("Use qmlRegisterAnonymousType instead") int qmlRegisterType()
      |                                                                          ^~~~~~~~~~~~~~~
# github.com/muesli/telephant
./telephant.go:142:23: too many arguments in call to gap.NewScope
        have (gap.ScopeType, string, string)
        want (gap.ScopeType, string)

==> ERROR: A failure occurred in build().
    Aborting...
```

**Comment by muesli** [comment]
Can you try again? I think this may have been resolved by both upgrading to Qt 5.14.2 and a recent fix in telephant I just pushed.

**Comment by bionade24** [comment]
Works now, thanks.
